### DISCUSSION TITLE
Requesting official support OpenAI's new gpt-4o model

### DISCUSSION THREAD

**Discussion by varenc** [langfuse/langfuse#2045] [Ideas]
### Describe the feature or potential improvement

OpenAI released their gpt-4o model today and it's now immediately available to all API users.  It make sense for Langfuse to include this as one of the official pre-filled models.  [The docs say](https://langfuse.com/docs/model-usage-and-cost#:~:text=model%20definitions%20or-,request%20official%20support%20for%20new%20models%20via%20GitHub.,-Ingested%20usage%20and) to request official support via an issue like this.

### Additional information

There's both a `gpt-4o` and a `gpt-4o-2024-05-13` per the usual convention.  Also I believe langfuse will need to update to the latest version 7.0 of `tiktoken` which was just updated with gpt-4o support.  

Here's a screenshot of the working a model definition I create for myself:
<img width="350" alt="image" src="https://github.com/langfuse/langfuse/assets/167585/b69705df-50a7-4109-8eab-fa8889a63203">

But note: in the model's openai Tokenizer config I had to just claim the model was the older gpt-4-turbo. Setting `{ ..., "tokenizerModel":"gpt-4o-2024-05-13", ... }` resulted in gpt-4o logs showing 0 tokens, hence the need for a `tiktoken` upgraded.

**Comment by marcklingen** [comment]
Thanks for posting this, we are currently adding the pricing information in a patch release to make this available for everyone. There won't be a change if you already added the price manually.

gpt4o uses a new tokenizer (`o200k_base` instead of `cl100k_base`, [diff of tiktoken](https://github.com/openai/tiktoken/commit/9d01e5670ff50eb74cdb96406c7f3d9add0ae2f8)). We run [js-tiktoken](https://github.com/dqbd/tiktoken) and will bump the dependency as soon as the new tokenizer is available and will update the model config and tokenization tests accordingly.

**Comment by marcklingen** [comment]
js-tiktoken was just updated, I'll include token counts in this patch release already: https://github.com/dqbd/tiktoken/releases/tag/js-tiktoken%401.0.12

**Comment by marcklingen** [comment]
@varenc released here by the way: https://github.com/langfuse/langfuse/releases/tag/v2.39.0

**Comment by clemra** [comment]
Hey folks, also working on getting the new model into Playground and Evals asap :) Will report back here

**Comment by clemra** [comment]
Here we go: https://langfuse.com/changelog/2024-05-14-openai-gpt-4o-playground-evals
